docs/migrations/web-react/MIGRATION-v2.md

```diff
@@ -256,6 +256,15 @@ See [Codemods documentation][readme-codemods] for more details.
 
 Or manually add `isDockedOnMobile` prop to the `ModalDialog` component.
 
+### TextField: `label` prop
```

Comment:
Please, update also the Table Of Contents at the beginning of this file.

Reply:
table updated